src/main/java/io/netty/incubator/codec/http3/Http3.java

```diff
@@ -57,6 +57,22 @@ public static QuicStreamChannel getLocalControlStream(Channel channel) {
         return channel.attr(HTTP3_CONTROL_STREAM_KEY).get();
     }
 
+    /**
+     * Returns the value of the <a
+     * href="https://quicwg.org/base-drafts/draft-ietf-quic-http.html#name-max_push_id"> max push ID</a> received for
```

Comment:
nit: consider linking to the RFC: https://www.rfc-editor.org/rfc/rfc9000.html

Reply:
I think I will come back and fix all links in code to link to the rfc, let me be consistent with other references for now.